Source/Orts.Simulation/Simulation/RollingStocks/MSTSWagon.cs

```diff
@@ -2296,11 +2296,16 @@ private void UpdateTrainBaseResistance_DavisLowSpeed()
             // Determine Axle loading of Car
             if (WagonType == WagonTypes.Engine && IsPlayerTrain && Simulator.PlayerLocomotive is MSTSLocomotive locoParameters)
             {
+                // This only takes into account the driven axles for 100% accuracy the non driven axles should also be considered
                 AxleLoadKg = locoParameters.DrvWheelWeightKg / locoParameters.LocoNumDrvAxles;
             }
             else
             {
-                AxleLoadKg = MassKG / WagonNumAxles;
+                // Typically this loop should only be processed when it is a car of some descritption, and therefore it will use the wagon axles as it reference.
```

Comment:
```suggestion
                // Typically this loop should only be processed when it is a car of some description, and therefore it will use the wagon axles as its reference.
```

Reply:
Thanks for identifying the typo, now fixed.